cdc/scheduler/internal/tp/scheduler_balance.go

```diff
@@ -28,6 +28,9 @@ type balanceScheduler struct {
 	random               *rand.Rand
 	lastRebalanceTime    time.Time
 	checkBalanceInterval time.Duration
+	// keepBalance if the last time schedule tasks is not empty.
```

Comment:
please let the comment be more clear, what does it indicate `if the last time schedule tasks is not empty` ?